packages/sui-svg/package.json

```diff
@@ -10,16 +10,16 @@
     "sui-svg": "./bin/sui-svg.js"
   },
   "dependencies": {
-    "@babel/core": "7.3.3",
-    "@compositor/kit": "1.0.47",
+    "@babel/core": "7.6.4",
     "@s-ui/bundler": "5",
     "@s-ui/component-peer-dependencies": "latest",
     "@s-ui/helpers": "1",
+    "@s-ui/react-atom-icon": "1",
     "@svgr/core": "3.1.0",
-    "commander": "2.19.0",
-    "fast-glob": "2.2.4",
-    "lodash.camelcase": "4.3.0",
-    "styled-components": "4.1.1"
```

Comment:
👋🏽😭

Reply:
hahaha It was only used internally by composer. :P Nothing to see here! Keep walking! 🚶